### PR TITLE
Release changelog for 2.17

### PR DESCRIPTION
**Pull request by sergiusens**
Signed-off-by: Sergio Schvezov <sergio.schvezov@ubuntu.com>

### CODE REVIEW

**Comment by sergiusens** [comment]
https://launchpad.net/~snappy-dev/+archive/ubuntu/snapcraft-daily/+build/10730034

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,31 @@
+snapcraft (2.17) xenial; urgency=medium
+
+  [ Sergio Schvezov ]
+  * Python plugin improvements (#771)
+  * Organize without removing files on target (#780)
+  * Support globbing for organize (#784)
```
Comment (kyrofa) [inline debian/changelog]: Missing some bugs on these three.

Comment (kyrofa) [inline debian/changelog]: Actually, bugs seem to be missing for all of these.

Comment (kyrofa) [inline debian/changelog]: Ah, I understand now, just the SRU bugref.

**Comment by kyrofa** [comment]
Looks good to me :+1: . Though I still wonder if leaving the debian changelog containing the SRU and maintaining a separate CHANGELOG containing the actual changes (and bugrefs) would be better.

**Comment by sergiusens** [comment]
El 09/09/16 a las 17:43, Kyle Fazzari escribió:
>
> Looks good to me 👍 . Though I still wonder if leaving the debian 
> changelog containing the SRU and maintaining a separate CHANGELOG 
> containing the actual changes (and bugrefs) would be better.
>
Yeah, we just need time, we also need to split out the packaging branch 
from master

**Comment by sergiusens** [comment]
https://launchpad.net/~snappy-dev/+archive/ubuntu/snapcraft-daily/+build/10730098

**Comment by elopio** [comment]
These are the tests to run for the SRU of this release:
https://gist.github.com/elopio/a749810edd9a80517b46ace63d45b166#file-snapcraft-tests-2-17-md

**Pull request merged**